cmd/create/create.go

```diff
@@ -0,0 +1,125 @@
+package create
```

Comment:
for thread/ reduce top comments?

Reply:
> _It would be nice for `I create` to send the name to stdout for better scripting._

Our current logging system is not quite friendly to scripts: https://github.com/iterative/terraform-provider-iterative/pull/423#pullrequestreview-901140497 but yes, that's a good idea; I'll hack something together.

Reply:
* https://github.com/iterative/terraform-provider-iterative/pull/610/commits/e4a01d4905993b1731a4396db0fc1ee967ed4092 causes `I` to complain only for configuration parsing errors, and be silent otherwise.
* https://github.com/iterative/terraform-provider-iterative/pull/610/commits/aa0c66e1f19f7cb35ee95fae0c955020e41a038a prints the task name to standard output using a lowly `fmt.Println` 😄